README.rst

```diff
@@ -28,6 +28,12 @@ Results of the evaluation (see below for reproduction and details)::
     readability          precision=0.913 ± 0.014  recall=0.931 ± 0.016  F1=0.922 ± 0.014 accuracy=0.315 ± 0.035
     xpath-text           precision=0.246 ± 0.016  recall=0.992 ± 0.001  F1=0.394 ± 0.020 accuracy=0.000 ± 0.000
 
+Result of packages added after original evaluation::
```

Comment:
I wonder if trafilatura should be also added to the first paragraph of README ("Article extraction benchmark: open-source libraries and commercial services" section)

Reply:
thanks for the catch @kmike , done in f5f76dc